### PR TITLE
Includes sensitive flag in yaml/json output of porter explain.

### PR DESCRIPTION
**Pull request by MadhuMPandurangi**
# What does this change

_Includes sensitive flag in yaml/json output of porter explain for outputs and parameters._

# What issue does it fix
Closes # _[2228](https://github.com/getporter/porter/issues/2228)_

# Notes for the reviewer
_Please verify the pull request._

# Checklist
- [ ] Did you write tests?
- [ ] Did you write documentation?
- [ ] Did you change porter.yaml or a storage document record? Update the corresponding schema file.
- [x] If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️

# Reviewer Checklist
* Comment with /azp run test-porter-release if a magefile or build script was modified
* Comment with /azp run porter-integration if it's a non-trivial PR

[contributors]: https://getporter.org/src/CONTRIBUTORS.md

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -74,3 +74,4 @@ and we will add you. **All** contributors belong here. 💯
 * [Kevin Barbour](https://github.com/kevinbarbour)
 * [Epsxy](https://github.com/epsxy)
 * [Jens Arnfast](https://github.com/jarnfast)
+* [Madhu M Pandurangi](https://github.com/MadhuMPandurangi)
```
Comment (carolynvs) [inline CONTRIBUTORS.md:77]: 🎉

**Review by carolynvs — CHANGES_REQUESTED** [review]
Let's change one of the parameters in the test bundle so that it's sensitive. Right now they are all false, so it's hard to tell that the change is working.

The test bundle is here: https://github.com/getporter/porter/blob/3a56d91ed05bf3a0d5a967bb3ad67685189d4b3c/pkg/porter/testdata/explain/params-bundle.json#L25

In order to make a parameter in a CNAB bundle sensitive (this is a cnab bundle here, you can tell because it's a json file, not porter.yaml), is to set `"writeOnly": true` in the parameter's definition.

```json
    "seed": {
      "type": "boolean"
      "writeOnly": true
    },
```

Hopefully we can just edit the existing parameter instead of adding another, but if it doesn't work well (maybe impacts other tests) then let's just add a new parameter that is sensitive.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (MadhuMPandurangi) [reply CONTRIBUTORS.md:77]: Thanks for adding. I'll look into the edits that need to be done in this.

**Comment by carolynvs** [comment]
In the future, if you can provide a copy/paste of the text, instead of an image, and include the full output, that would be helpful. It looks like you did not update the expected output files for the test that failed. The tests are still asserting that none of the parameters are sensitive.

You can either update the expected output files by hand, or run `mage UpdateTestFiles` and have it updated to automatically match whatever the current output is from that test.

**Comment by MadhuMPandurangi** [comment]
hi @carolynvs, I added "writeonly": true in the parameter definition, and after updating the test cases, all the test cases are successfully passed.

**Review by carolynvs — APPROVED** [review]
Looks great! Again apologies for the delay with the review, I appreciate your time getting this implemented. 💖

**Comment by MadhuMPandurangi** [comment]
Thank you @carolynvs . This is my first contribution to the opensource and happy to see merged 😊.

**Comment by carolynvs** [comment]
I'm looking forward to collaborating with you in the future!

**Pull request merged**